### DISCUSSION TITLE
基于关系感知路由的全球流量调度 · SOSP 2019 - 面向信仰编程 · /papers-taiji

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#627] [Announcements]
https://draveness.me/papers-taiji/

**Comment by haosdent** [comment]
这个很像阿里之前的单元化, 按照买家用户mod到不同idc.我们的搞法则是直接GeoIP

**Comment by draveness** [comment]
> @haosdent 这个很像阿里之前的单元化, 按照买家用户mod到不同idc.我们的搞法则是直接GeoIP

不过阿里的单元化没有考虑不同用户之间的社交关系

**Comment by gaocegege** [comment]
Facebook 的 paper 感觉有个特点，就是跟业务太相关，跟之前的 TAO 等等一脉相承
（粉丝留名

**Comment by draveness** [comment]
> @gaocegege
> Facebook 的 paper 感觉有个特点，就是跟业务太相关，跟之前的 TAO 等等一脉相承
> （粉丝留名

毕竟是做社交网络的，不过也有很多与具体业务无关的工作，比如 RocksDB 相关的，从这里能找到很多有意思的工作 https://research.fb.com/publications/

**Comment by leeweir** [comment]
> @gaocegege
> Facebook 的 paper 感觉有个特点，就是跟业务太相关，跟之前的 TAO 等等一脉相承
> （粉丝留名

策哥真的无处不在
